components/followList/FollowListItem.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import User from 'types/userTypes';
+
+import { COLOR } from '@/styles/colors';
+
+import Checkbox from './CheckBox';
+
+interface FollowListItemProps {
+  user: User;
+  avatar_url: string;
+  showCheckbox: boolean;
+}
```

Comment:
요것도 types에 선언해놓은 User 사용한다면
```suggestion
interface FollowListItemProps extends User {
  showCheckbox: boolean;
}
```